targets/TARGET_STM/rtc_api.c

```diff
@@ -341,11 +334,16 @@ static void RTC_IRQHandler(void)
 
 uint32_t rtc_read_lp(void)
 {
+    /* LPTICKER time is reading only time RTC register (in second) and add the sub-second part
+    *  In order to get a continuous time information (each 24h = 86400s = 0x15180),
+    *  current time is added into an internal counter : LP_continuous_time
+    *  In order to get spent time since last counter update, current RTC time is saved : LP_last_RTC_time
+    */
     struct tm timeinfo;
+    core_util_critical_section_enter();
```

Comment:
Was something calling this outside a critical section?

Reply:
I added protection, as rtc_write and rtc_read_lp functions can update the same variables